assets/templates/release.tmpl

```diff
@@ -1,4 +1,15 @@
-<div class="ons-page__container ons-container release">
+<div class="ons-page__container ons-container release" 
+  {{ if .Description.Published }}
+    data-gtm-release-status="published"
+  {{ else }}
+    data-gtm-release-status="cancelled"
+  {{ end }}
+  data-gtm-release-date="{{dateFormatYYYYMMDDNoSlashes .Description.ReleaseDate}}"
+  data-gtm-release-time="{{dateFormatYYYYMMDDNoSlashes .Description.ReleaseDate }}"
```

Comment:
I presume this call to `dateFormatYYYYMMDDNoSlashes` will change to the new `TimeFormat` once the new dp-renderer is imported as part of this PR.

Reply:
This is now done